Nos/Views/AuthorStoryView.swift

```diff
@@ -192,23 +192,9 @@ fileprivate struct BottomOverlay: View {
 
     @EnvironmentObject private var router: Router
 
-    @State private var replyCount = 0
-    @State private var replyAvatarURLs = [URL]()
-
     var body: some View {
         HStack(spacing: 0) {
-            StackedAvatarsView(avatarUrls: replyAvatarURLs, size: 20, border: 0)
-                .padding(.trailing, 8)
-
-            if let replies = attributedReplies {
-                Button {
-                    router.push(note)
-                } label: {
-                    Text(replies)
-                        .font(.subheadline)
-                        .foregroundColor(Color.secondaryTxt)
-                }
-            }
+            ReplyCountLabel(replyCount: .discussion, for: note)
```

Comment:
I think I'd prefer this:

```suggestion
            RepliesLabel(for: note, displayType: .discussion)
```

To me, that's a little easier to read and understand.